packages/core/src/Editor.ts

```diff
@@ -80,6 +81,7 @@ export class Editor extends EventEmitter<EditorEvents> {
     enablePasteRules: true,
     enableCoreExtensions: true,
     enableContentCheck: false,
+    addTypeAttributes: true,
```

Comment:
Can we make this default `false` for backwards-compat?

v3 we can do it default on

Reply:
done